app/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
+    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
```

Comment:
Why are you using this? You can use the one from AppCompat. Does this offer anything extra?